.github/CODEOWNERS

```diff
@@ -1,3 +1,5 @@
+* @hashicorp/consul-selfmanage-maintainers
```

Comment:
Should we add copyright information to this file? 
```
# Copyright (c) HashiCorp, Inc.
# SPDX-License-Identifier: MPL-2.0```

Reply:
I checked the consul repo, and didn't see it for codeowners so I think it's ok without? None of the other codeowners files seem to have it either 🤔